spec/music_album_spec.rb

```diff
@@ -0,0 +1,43 @@
+require './genre'
+require './music_album'
+require 'date'
+
+describe MusicAlbum do
+  let(:genre) { Genre.new('Rock') }
+
+  context 'when receiving valid parameters' do
+    let(:music_album) { MusicAlbum.new(genre, '2012-07-10', true) }
+
+    it 'returns a valid MusicAlbum object' do
+      expect(music_album).to be_instance_of(MusicAlbum)
+    end
+  end
```

Comment:
- [ ] One of the project requirements is that all public methods in all classes should have unit tests. Please write unit tests for the `to_hash` method in your `MusicAlbum` class 👍